### PR TITLE
sync content/zh/docs/reference/issues-security/ en zh

### PR DESCRIPTION
**Pull request by tanjunchen**
@tengqm

### CODE REVIEW

**Comment by tanjunchen** [comment]
/hold
for show problems

**Comment by tanjunchen** [comment]
it is  similar with https://github.com/kubernetes/website/issues/18737 , so i think we can first sync zh en,
and resovling this problems.
/cc @tengqm 
keeping consistent with en/ is more important than this. what do you  think?
/hold cacel

File: content/zh/docs/reference/issues-security/security.md [hunk 1]
```diff
@@ -36,6 +34,11 @@ Join the [kubernetes-announce](https://groups.google.com/forum/#!forum/kubernete
 -->
 加入 [kubernets-announce](https://groups.google.com/forum/#!forum/kubernetes-announce) 组，以获取关于安全性和主要 API 公告的电子邮件。
 
+<!--
+You can also subscribe to an RSS feed of the above using [this link](https://groups.google.com/forum/feed/kubernetes-announce/msgs/rss_v2_0.xml?num=50).
+-->
+您也可以使用[该链接](https://groups.google.com/forum/feed/kubernetes-announce/msgs/rss_v2_0.xml?num=50)订阅上述的 RSS 反馈。
```
Comment (tengqm) [inline content/zh/docs/reference/issues-security/security.md]:
```suggestion
您也可以使用[此链接](https://groups.google.com/forum/feed/kubernetes-announce/msgs/rss_v2_0.xml?num=50)订阅上述的 RSS 反馈。
```

File: content/zh/docs/reference/issues-security/security.md [hunk 2]
```diff
@@ -66,11 +73,13 @@ You may encrypt your email to this list using the GPG keys of the [Product Secur
...
 - You are unsure how a vulnerability affects Kubernetes
-- You think you discovered a vulnerability in another project that Kubernetes depends on (e.g. docker, rkt, etcd)
+- You think you discovered a vulnerability in another project that Kubernetes depends on
+   - For projects with their own vulnerability reporting and disclosure process, please report it directly there
 -->
 - 您认为在 Kubernetes 中发现了一个潜在的安全漏洞
 - 您不确定漏洞如何影响 Kubernetes
-- 您认为您在 Kubernetes 依赖的另一个项目中发现了一个漏洞（例如 docker、rkt、etcd）
+- 您认为您在 Kubernetes 依赖的另一个项目中发现了一个漏洞
+  - 对于具有漏洞报告和披露流程的项目，请直接在此处报告
```
Comment (tengqm) [inline content/zh/docs/reference/issues-security/security.md]:
```suggestion
  - 对于具有漏洞报告和披露流程的项目，请直接在该项目处报告
```

Comment (tanjunchen) [reply content/zh/docs/reference/issues-security/security.md]: thanks your review , i will update it

File: content/zh/docs/reference/issues-security/issues.md [hunk 3]
```diff
@@ -1,16 +1,33 @@
 ---
 title: Kubernetes 问题追踪
 weight: 10
+aliases: [/cve/,/cves/]
```
Comment (zacharysarah) [inline content/zh/docs/reference/issues-security/issues.md]: @tanjunchen This PR reintroduced the bug in #18691. 

Aliases in localized versions need the leading slash removed. These should instead be:
```
aliases: [cve/, cves/]
```

File: content/zh/docs/reference/issues-security/security.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Kubernetes 安全和信息披露
+aliases: [/security/]
```
Comment (zacharysarah) [inline content/zh/docs/reference/issues-security/security.md]: This should also be:
```
aliases: [security/]
```

Comment (tanjunchen) [reply content/zh/docs/reference/issues-security/security.md]: sure 
Thanks , i get it

File: content/zh/docs/reference/issues-security/issues.md (hunk 3 above)
Comment (tanjunchen) [reply content/zh/docs/reference/issues-security/issues.md]: ok

**Pull request merged**